.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+# Steps:
+# 1. Build the xrc
+# 2. Push the release out.
+
+name: "release"
+
+on:
+  push:
+    tags:
+      - "*"
+
+env:
+  DFX_VERSION: "0.14.1"
```

Comment:
There are some issues with this version, right?
Would it make sense to wait for the 0.14.2 release or is this not necessary?

Reply:
It's not necessary. I believe 0.14.2 has been released. I can bump the version in a separate PR.